### DISCUSSION TITLE
Is there something like breadcrumbs available?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#9981] [Q&A]
https://wordpress.org/plugins/breadcrumb/

**Comment by hishamco** [comment]
Breadcumb is available in the admin theme, are you looking for breadcrumb component in both admin & site theme?

**Reply by xperiandri** [reply]
We need it in the site theme

**Reply by hishamco** [reply]
We could expose it as component to be reusable in both admin & site theme. @agriffard is a breadcrumb component right now or added as markup? If No, we could add it as a navigation component

**Reply by Diman96** [reply]
Could you say in which file in admin theme breadcumb is used?

**Reply by hishamco** [reply]
The default admin theme

**Reply by xperiandri** [reply]
Could you give a link to the file and the line

**Comment by deanmarcussen** [comment]
There is no admin theme breadcrumb component. just a Zone.

@Piedone wrote a blog article a while ago about building a breadcrumb from a menu. Maybe he will relink here for you

**Reply by Piedone** [reply]
Sure: https://orcharddojo.net/blog/how-to-add-a-breadcrumb-menu-in-orchard-core-orchard-nuggets
